src/luaopenssl/lua_openssl.cc

```diff
@@ -275,20 +267,19 @@ namespace {
 	* @return OpenSSL BIO structure
 	**/
 	BIO* newBIO(const int type) {
-		BIO_METHOD* methodType;
+		BIO* bio;
```

Comment:
@dboltovskyi  probably should be initialized with `null` value

Reply:
If it works, for sure :)
```suggestion
		BIO* bio = null;
```